hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/otus-hw/hw01_hello_otus
```

```suggestion
module github.com/Vadim-Govorukhin/otus-hw/hw01_hello_otus
```
Рекомендую указать полный путь до модуля